packages/manager/modules/metrics/src/token/preview/metrics-token-preview.less

```
@@ -1,5 +1,6 @@
 .metrics__token-preview  {
   @labels-color: #676767;
+  @oui-color-concrete: #f2f2f2;
```

Maybe we could import ui-kit variables instead ?

---

I dont think i can import only variables. Do i have to import from dist ?

---

If `ovh-ui-kit` is defined as a peerDep, it is possible 
something like 
```suggestion
  @import ''ovh-ui-kit/packages/oui-color/_variables';
```
or something like that. 
Imo it would better but it is not mandatory to merge this pull request, this could be refactored later